artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -1,322 +1,191 @@
-= ArgoCD plugin for {product}
+= Installation and Configuration of ArgoCD
```

Comment:
Can we use the name to Argo CD with a space throughout this doc?  Even in backstage-plugins repo, there is a PR to update the naming with a space. 

```suggestion
= Installation and Configuration of Argo CD plugin
```